caddyhttp/proxy/reverseproxy.go

```diff
@@ -254,6 +254,10 @@ func (rp *ReverseProxy) copyResponse(dst io.Writer, src io.Reader) {
 
 func copyHeader(dst, src http.Header) {
 	for k, vv := range src {
+		// some predefined headers (e.g. MIME plugin), skip.
+		if _, ok := dst[k]; ok {
+			continue
+		}
```

Comment:
Yes, it definitely makes sense. But what kind of fields should be skipped?
Also I think the expected result should be like this: suppose we set `Content-Type = text/css` with MIME plugin and upstream responses with `Content-Type = text/html, encoding=utf8`, the expected result should be `Content-Type = text/css, encoding=utf8` (just combination).

Above is just for `Content-Type`, what about other header fields?